.github/workflows/kubernetes-ci.yml

```diff
@@ -68,6 +68,20 @@ jobs:
           echo 'KUBERNETES_CLIENT_TOKEN='"${KUBERNETES_CLIENT_TOKEN_CONTENT}"
           echo 'KUBERNETES_CLIENT_TOKEN_FILE='${KUBERNETES_CLIENT_TOKEN_FILE}
 
+          openssl genrsa -out ./t/certs/k8s_mtls.key 4096
+
+          openssl req -new -key ./t/certs/k8s_mtls.key -config ./t/certs/k8s_mtls_csr.conf -out ./t/certs/k8s_mtls.csr -nodes
+
+          export BASE64_CSR=$(cat ./t/certs/k8s_mtls.csr | base64 | tr -d '\n')
+
+          envsubst < ./t/certs/k8s_mtls_csr.yaml | kubectl apply -f -
```

Comment:
It would be better to set up a mtls proxy in the CI. Pros:
1. don't need to touch current k8s config
2. don't need to add cert config in non-mtls test
3. we can cover non-mtls path in the test
4. we can test different cert without duplicate k8s env